pkg/pci/gen.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build ignore
+// +build ignore
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strconv"
+
+	"github.com/u-root/u-root/pkg/pci"
+)
+
+var (
+	pciidspath = [...]string{"/usr/share/misc/pci.ids"}
```

Comment:
Is this expected to be present on the machine of build ?

Reply:
it's a widely distributed, slowly changing but note: there is a supplied pci_ids.go, checked in, which is a common practice with generated files. So if the generate does not work, you still have the pci_ids.go you started with. 

That's how we've done it for about 7 years ... believe it or not! I did not at first! It was my fault.